### PR TITLE
k8s: fix node to host link handling

### PR DESCRIPTION
**Pull request by lebauce**

### CODE REVIEW

File: agent/host.go [hunk 1]
```diff
@@ -54,7 +55,11 @@ type CPUInfo struct {
 // createRootNode creates a graph.Node based on the host properties and aims to have an unique ID
 func createRootNode(g *graph.Graph) (*graph.Node, error) {
 	hostID := config.GetString("host_id")
-	m := graph.Metadata{"Name": hostID, "Type": "host"}
+	hostname, err := os.Hostname()
+	if err != nil {
+		return nil, err
```
Comment (hunchback) [inline agent/host.go]: I rather you set `hostname = ""` and continue adding the root node, would make it more robust

File: topology/probes/k8s/node.go [hunk 2]
```diff
@@ -116,9 +116,20 @@ func (c *nodeProbe) OnDelete(obj interface{}) {
 	}
 }
 
+func (c *nodeProbe) OnNodeAdded(n *graph.Node) {
+	hostname, _ := n.GetFieldString("Hostname")
+	if t, _ := n.GetFieldString("Type"); t == "host" {
```
Comment (hunchback) [inline topology/probes/k8s/node.go]: this condition is redundant, your indexer only triggers events when `"Type" == "host"`

File: agent/host.go (hunk 1 above)
Comment (lebauce) [reply agent/host.go]: We already `panic` - on purpose - if we can't retrieve the hostname in config/config.go
If we can't get the hostname, I'm pretty sure nothing will work :-)

File: topology/probes/k8s/node.go (hunk 2 above)
Comment (lebauce) [reply topology/probes/k8s/node.go]: It was a precaution but a useless one. Removed

File: agent/host.go (hunk 1 above)
Comment (hunchback) [reply agent/host.go]: fine

**Pull request merged**